hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -874,7 +874,7 @@
 
 <property>
   <name>dfs.datanode.directoryscan.throttle.limit.ms.per.sec</name>
-  <value>1000</value>
+  <value>-1</value>
```

Comment:
This is not just a doc change, This change will indeed change the default.

Reply:
yep, here is the realated JIRA:HDFS-13947 @ayushtkn

Reply:
@ayushtkn  any new ideas?